nxc/protocols/ldap.py

```diff
@@ -1050,7 +1030,7 @@ def kerberoasting(self):
                         self.logger.debug(f"Exception: {e}", exc_info=True)
                         self.logger.fail(f"Principal: {downLevelLogonName} - {e}")
                 else:
-                    self.logger.fail(f"Error retrieving TGT for {self.username}\\{self.domain} from {self.kdcHost}")
```

Comment:
This is the actual change. Maybe ruff hadn't been run on this file before, but I ran `uvx ruff check --fix` which should have abided by the ruff config in pyproject.toml...